droidmediacodec.h

```diff
@@ -102,7 +102,7 @@ DroidMediaBufferQueue *droid_media_codec_get_buffer_queue (DroidMediaCodec *code
 DroidMediaCodec *droid_media_codec_create_decoder(DroidMediaCodecDecoderMetaData *meta);
 DroidMediaCodec *droid_media_codec_create_encoder(DroidMediaCodecEncoderMetaData *meta);
 bool droid_media_codec_is_supported(DroidMediaCodecMetaData *meta, bool encoder);
-unsigned int droid_media_codec_get_supported_color_formats(const char *mime, int encoder,
+unsigned int droid_media_codec_get_supported_color_formats(DroidMediaCodecMetaData *meta, int encoder,
```

Comment:
The signature of the wrapper for `droid_media_codec_get_supported_color_formats()` also needs to be changed in hybris.c.